Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -295,6 +295,27 @@ Advanced Configuration
 This section covers a few advanced configuration modes for the kube-proxy replacement
 that go beyond the above Quick-Start guide and are entirely optional.
 
+Policy Enforcement
+******************
+
+When accessing a service from outside the cluster, the :ref:`arch_id_security`
+assignment depends on the routing and the kube-proxy replacement configuration.
+
+In the tunneling mode (i.e., ``--tunnel=vxlan`` or ``--tunnel=geneve``), the request
+to the service will have the ``reserved:world`` security identity. This allows a user to
+use a network policy with the ``fromEntities: cluster`` label selector.
+
+In the direct-routing mode (i.e., ``--tunnel=disabled``), the security identity
+will be set to ``reserved:world`` if the request was sent to the node which runs the
+selected endpoint by the LB. If not, i.e., the request needs to be forwarded to
+another node after the service endpoint selection, then it will have either ``reserved:host``
+or ``reserved:remote-node``. This makes the enforcement of the network policy
+inconsistent.
```

Comment:
I'd be tempted to just say that the identity of the traffic is undefined for this case instead of giving this selection of "it could be {world,host,remote-node}". It's clear that if the identity is inconsistent here, then this was never designed to work with network policies in any sort of sane way. 

Though that said, it is actually important that the identity *can* be `remote-node` / `host`, as this means that `fromEntities: cluster` policies could actually match on this nodeport traffic incoming to the cluster, which would surprise almost everyone who attempts to use `fromEntities: cluster` policy.